endpoints-routes-v1.0.yml

```diff
@@ -162,3 +162,35 @@
     - code: 415
       reason: File not off correct mime. We only support .IGC
       content-type: application/json
+
```

Comment:
If any of the API specifications change, notify me so I can update swagger

Reply:
Well there are now 2 more paths.